pkg/app/config.go

```diff
@@ -100,8 +100,9 @@ func CreateConfig(configFile string) (Config, error) {
 			username = config.Registry[regCount].User
 			password = config.Registry[regCount].Pass
 		case "":
-			username = ""
-			password = ""
+			// Assuming that the user has either no credentials or defined them in the config
+			username = reg.User
+			password = reg.Pass
```

Comment:
Why isn't this like the code in `case "config"`?

```
username = config.Registry[regCount].User
password = config.Registry[regCount].Pass
```
Feels like `case ""` should be the same as `case "config"` without the validation of the credentials.